### PR TITLE
Protect privacy mv3 update

### PR DESCRIPTION
**Pull request by IanStanion-google**
See [#SOME_ISSUE_NUMBER](https://github.com/GoogleChromeLabs/Extension-Docs/issues/85)

### CODE REVIEW

File: site/en/docs/extensions/mv3/user_privacy/index.md [hunk 1]
```diff
@@ -57,23 +58,25 @@ features.
...
-    origins: ['https://www.google.com/']
-  }, function(granted) {
-    // The callback argument will be true if the user granted the permissions.
-    if (granted) {
-      // doSomething();
-    } else {
-      // doSomethingElse();
+  chrome.permissions.request(
+    {
+      permissions: ["tabs", "scripting"],
```
Comment (oliverdunk) [inline site/en/docs/extensions/mv3/user_privacy/index.md:69]: Did we drop `origins` here intentionally?

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Nice simple one, looks good! Left one comment.

**Comment by jpmedley** [comment]
@oliverdunk Are you done with this?

**Comment by oliverdunk** [comment]
> @oliverdunk Are you done with this?

Yep, this looks good to me. LGTM.

**Review by oliverdunk — APPROVED** [review]

File: site/en/docs/extensions/mv3/user_privacy/index.md [hunk 2]
```diff
@@ -57,23 +58,26 @@ features.
 {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/5hIYQFOiYTl4EwyaYMLx.png",
        alt="A screenshot of a popup asking to enable permissions", height="248", width="258" %}
 
-Clicking **Okay!** will trigger the following event in the background script.
+Clicking **Okay!** will trigger the following event in the service worker.
 
 ```js
-document.querySelector('#button').addEventListener('click', function(event) {
+chrome.action.onClicked.addListener(function (event) {
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/user_privacy/index.md:64]:
```suggestion
chrome.action.onClicked.addListener((event) => {
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Pull request merged**